tests/endpoints/users/test_get_user_by_email.py

```diff
@@ -0,0 +1,83 @@
+import json
+import unittest
+from unittest.mock import patch
+from copy import deepcopy
+
+from api import create_app, db
+from api.database.models import User
+from tests import db_drop_everything, assert_payload_field_type_value, \
+    assert_payload_field_type
+
+
+class GetUserTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
This should be snake case instead of camel case.

Reply:
Fun discovery of unittest syntax! :P